lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
I looked at `Pathname` and didn't find a useful method for this check. `Pathname#relative_path_from` looked promising but didn't end up offering a way to enforce that a path is a child of a base directory.

For example:

```ruby
>> base = Pathname.new(Dir.pwd)
>> Pathname.new("/tmp/example.txt").relative_path_from(base)
=> #<Pathname:../../../../tmp/example.txt>